openai/release_notes.md

```diff
@@ -1,9 +1,4 @@
 #### What's Improved
-- Added following new actions:
-  - Converse With OpenAI
-  - List Available Models
-  - Get Tokens Usage
-  - Get Token Count
-- Renamed the `Create a chat completion` action and playbook to `Ask a Question`.
-- Added `Timeout` parameter in `Ask a Question` action.
-- Enhanced connector to be able to connect to Azure OpenAI Service. A new configuration parameter named 'Use Microsoft Azure OpenAI Endpoint' is added for users who want to use OpenAI services via a Microsoft Azure Endpoint. 
\ No newline at end of file
+- Added a new configuration parameter `Verify SSL`.
+- Initialized httpx client with proxy if proxy is configured in the environment.
```

Comment:
Added Network Proxy support. Call to openai endpoint will be routed via the network proxy if proxy is configured for the FortiSOAR instance